db/anonymise/rules.rb

```diff
@@ -0,0 +1,144 @@
+require 'faker'
+
+NINO_REGEXP = /^[A-CEGHJ-PR-TW-Z]{1}[A-CEGHJ-NPR-TW-Z]{1}[0-9]{6}[A-DFM]{1}$/.freeze
+
+{
+  # data is set by us and should not leak data
+  # active_storage_attachments,
+  # active_storage_blobs
+  # active_storage_variant_records
+  # actor_permissions
+  # application_proceeding_types
+  # application_proceeding_types_scope_limitations
+  # application_scope_limitations
+  # ar_internal_metadata
+  # attachments
+  # bank_holidays
+  # benefit_check_results
+  # ccms_submission_documents
+  # ccms_submissions
+  # cfe_results
+  # cfe_submission_histories
+  # cfe_submissions
```

Comment:
should these 3 cfe entries be removed as we do anonymise some content on these tables